content/02.introduction.md

```diff
@@ -3,42 +3,39 @@
+This genomic information, coupled with functional genomics data on the transcriptome, epigenome and metabolome, has been touted to further drive precision medicine approaches using genetics.
 
-Complementing the rapid growth in our understanding of gene variants in the human genome was the emergence of using statistical techniques, formalized as genome-wide association studies (GWAS), to identify gene variants associated with common human diseases.
-From a population perspective, GWA studies have sought to discern genetic connections to various phenotypes by studying genotypic variation at biallelic markers across the human genome [@doi:10.1371/journal.pcbi.1002822; @doi:10.1038/nrg1521; @doi:10.1038/nrg1522].
-Such non-candidate driven GWA studies consider gene variations (i.e. SNPs, deletions, intertions, CNVs) to resulting phenotype values to ultimately report allele frequency differences among a case and control group in the form of an odds ratio.
-This technical revolution in the field of genomic medicine fueled our progressing capabilities to map associations of gene variants with disease on an increasingly granular level to single nucleotide polymorphisms (SNPs). 
+Complementing the rapid growth in our understanding of human genetic variation was the emergence of using statistical techniques, formalized as genome-wide association studies (GWAS) in the early 2000s, to identify gene variants associated with common human diseases.
+From a population perspective, GWAS have sought to discern genetic connections to various phenotypes by studying genotypic variation at single nucleotide polymorphisms (SNPs) across the human genome [@doi:10.1371/journal.pcbi.1002822; @doi:10.1038/nrg1521; @doi:10.1038/nrg1522].
 
 In tandem with the movement towards precision medicine, the post-GWAS era strives to bring significant population-derived gene variants into individual level metrics actionable in health delivery settings.
-While GWA studies indeed capture gene variants associated with a phenotype of interest on a population level, translating such results to personalized individual metrics of risk requires additional granularity on aggregating contributions of many gene variants in the form of polygenic risk scores (PRS).
-Importantly, PRS provides an ability to explain inherited risk for disease in an individual by representing a weighted sum aggregate of risk alleles based on measured loci effect contributions derived from GWAS studies [@doi:10.1001/jama.2019.3893; @doi:10.1038/s41576-018-0018-x]. 
+While GWAS indeed capture gene variants associated with a phenotype of interest on a population level, translating such results to personalized individual metrics of risk requires aggregating contributions of many gene variants in the form of polygenic risk scores (PRS).
```

Comment:
I think at I used PRSs somewhere in Methods. We need to check for this consistency later.